app/models/case/sar/offender.rb

```diff
@@ -185,7 +185,8 @@ def requester_address
   end
 
   def recipient_name
-    (!subject_recipient?) ? third_party_name : subject_name
+    return subject_name if subject_recipient?
```

Comment:
Is there anyway that both of these could be `nil`? I'm thinking that this would not be allowed be the model validations.